### ISSUE TITLE
highlight is under the text and not behind that

### ISSUE THREAD

**Issue by MattiaPontonioKineton** [WebPlatformForEmbedded/Lightning#86]
Highlight is under the text and not behind as espected.
Is the correct behaviour?
Code:
```
Text: {
                x: 0,
                y: 450,
                text: {
                    text: 'Text a little too long so this part won\'t be visible',
                    highlight:true,
                    highlightColor:0xFF557722
                }
            },
            Text2: {
                x: 150,
                y: 100,
                text: {
                    fontSize: 80,
                    text: 'STYLE IT UP! ',
                    fontStyle: 'italic bold',
                    textColor: 0xff00ffff,
                    shadow: true,
                    shadowColor: 0xffff00ff,
                    shadowOffsetX: 2,
                    shadowOffsetY: 2,
                    shadowBlur: 2,
                    highlight:true,
                    highlightColor:0xFF557722
                }
            },
            Text3: {
                mountX: 1,
                x: 530,
                y: 230,
                text: {
                    fontSize: 48,
                    textAlign: 'left',
                    text: 'you can also add\nline breaks\nusing\n\\n!',
                    lineHeight: 48,
                    highlight:true,
                    highlightColor:0xFF557722
                }
            },
```
![Immagine](https://user-images.githubusercontent.com/58947115/71979439-04d99f00-321e-11ea-8074-80b8bb102912.png)

**Comment by erikhaandrikman** [comment]
This is not correct behaviour, we have fixed in the following branch: 
https://github.com/WebPlatformForEmbedded/Lightning/tree/fix/text-highlight

Will notify you when it's in master

**Comment by g-zachar** [comment]
Fixed in 1.3.1

[example](https://jsfiddle.net/werchq0s/2/)
